Automattic/wp-calypso
# Site settings: modify the copy of the Disconnect Site component.

We are working towards improved usage of the disconnect flow data. As such, we modify and simplify the questions asked. 

#### Changes proposed in this Pull Request

- remove follow-up questions and related files
- change the content of the main screen
- remove now redundant routes
- start a new table for saving the data: `calypso-disconnect-jetpack-july2019`

#### Testing instructions

- start the flow at /settings/disconnect-site/:site, where :site is a connected Jetpack site,
- verify the Qs and headers:
<img width="509" alt="Screenshot 2019-07-26 at 17 23 05" src="https://user-images.githubusercontent.com/13561163/61962735-7ef8cd00-afca-11e9-9640-5a2911a8c092.png">



- test the routing by: go back, skip, chose an option, stay connected, disconnect.

- on pressing the final button Disconnect: 
Verify in the dev console that the survey data that would be submitted match what you specified in the survey ( submitSurvey isn't a Redux but a Flux action, watch the Network tab to make sure your data are actually sent).
